lib/child-writer/index.js

```diff
@@ -170,10 +172,35 @@ exports.write = (image, drive, options) => {
     }
   }
 
+  // Timeout function that only runs if we exceed the specified timeout,
+  // terminating the server.
+  let terminationTimeout = null
+
+  /**
+   * @summary Reset the timeout countdown
+   * @function
+   * @private
+   *
+   * @example
+   * resetTerminationTimeout()
+   */
+  const resetTerminationTimeout = () => {
+    clearTimeout(terminationTimeout)
+    terminationTimeout = setTimeout(() => {
+      emitter.emit('timeout', {
```

Comment:
I think we should also make sure we kill the child (if its still alive) if the timeout occurred.